### DISCUSSION TITLE
API KEY not found

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#193] [Help Me!] [answered]
Dani Crosby
Have you searched the other help tickets? Yess

Teammate pulled down from setup, but getting error code about an API not being found. We tried copy pasting all keys again from firebase. Code reads: 

{code: "auth/internal-error", message: "{"error":{"code":400,"message":"API key not valid.…son":"badRequest"}],"status":"INVALID_ARGUMENT"}}", a: null}

Links: 
https://firebase.google.com/docs/auth/admin/errors
https://github.com/firebase/firebaseui-web-react/issues/57
https://github.com/firebase/firebaseui-web-react

Re-pasted env files
stopped restarted server (3 times)

Link to GH Branch: https://github.com/nss-evening-cohort-14/pan-am-group-project/tree/development

Searched tickets #177 and #125
Still getting an error after changing the config

*** ANSWER *** We had commas after our keys. Once removed our keys worked!

**Comment by drteresavasquez** [comment]
Please reference this thread for all the info needed for tickets. Also, if you referenced other tickets with similar issues, please let us know that as well: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/2#discussion-48440

**Comment by TrinityChristiana** [comment, marked as answer]
They had commas after each key in the `.env` file.

**Comment by drteresavasquez** [comment]
@danicrosby and team GOOD FIND! I'm sure this will help someone in the future.
